custom_components/places/update_sensor.py

```
@@ -672,6 +672,18 @@ async def get_map_link(self) -> None:
         )
 
+    async def async_apply_show_time(self) -> None:
+        """Apply the configured last-updated suffix to existing sensor state."""
+        state = clear_since_from_state(self.coordinator.get_attr_safe_str(ATTR_NATIVE_VALUE))
+        if self.coordinator.get_attr(CONF_SHOW_TIME) and state:
+            now = await self.get_current_time()
+            suffix = f" (since {now.hour:02}:{now.minute:02})"
+            state = f"{state[: 255 - len(suffix)]}{suffix}"
+        else:
+            state = state[:255]
+        self.coordinator.set_attr(ATTR_SHOW_DATE, False)
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)